code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -76,54 +80,91 @@
 
 // ENGINEERING
 /obj/item/clothing/suit/space/plasmaman/assistant
+	name = "plasmaman assistant suit"
 	icon_state = "plasmamanAssistant_suit"
+	slowdown = 1
 
 /obj/item/clothing/head/helmet/space/plasmaman/assistant
+	name = "plasmaman assistant helmet"
 	icon_state = "plasmamanAssistant_helmet0"
 	base_state = "plasmamanAssistant_helmet"
 
 /obj/item/clothing/suit/space/plasmaman/atmostech
+	name = "plasmaman atmospheric suit"
 	icon_state = "plasmamanAtmos_suit"
+	armor = list(melee = 20, bullet = 0, laser = 0,energy = 0, bomb = 25, bio = 100, rad = 0)
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
 
 /obj/item/clothing/head/helmet/space/plasmaman/atmostech
+	name = "plasmaman atmospheric helmet"
 	icon_state = "plasmamanAtmos_helmet0"
 	base_state = "plasmamanAtmos_helmet"
+	armor = list(melee = 20, bullet = 0, laser = 0,energy = 0, bomb = 25, bio = 100, rad = 0)
+	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
 
 /obj/item/clothing/suit/space/plasmaman/engineer
+	name = "plasmaman engineer suit"
 	icon_state = "plasmamanEngineer_suit"
+	armor = list(melee = 40, bullet = 5, laser = 20,energy = 5, bomb = 35, bio = 100, rad = 80)
```

Comment:
You also might want to add the pressure_resistance of 200 * ONE_ATMOSPHERE (the standard for engies) to both this and helmet.